susemanager/src/mgr_bootstrap_data.py

```diff
@@ -774,10 +777,18 @@
         'BASECHANNEL' : 'opensuse_leap15_0-x86_64', 'PKGLIST' : PKGLIST15 + PKGLIST15_X86_ARM,
         'DEST' : '/srv/www/htdocs/pub/repositories/opensuse/15/0/bootstrap/'
     },
+    'openSUSE-Leap-15.1-x86_64-uyuni' : {
+        'BASECHANNEL' : 'opensuse_leap15_1-x86_64', 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15_X86_ARM,
+        'DEST' : '/srv/www/htdocs/pub/repositories/opensuse/15/1/bootstrap/'
+    },
     'openSUSE-Leap-15.1-x86_64' : {
-        'BASECHANNEL' : 'opensuse_leap15_1-x86_64', 'PKGLIST' : PKGLIST15 + PKGLIST15_X86_ARM,
+        'PDID' : [1929], 'PKGLIST' : PKGLIST15_SALT + PKGLIST15_X86_ARM,
         'DEST' : '/srv/www/htdocs/pub/repositories/opensuse/15/1/bootstrap/'
     },
+    'openSUSE-Leap-15.2-x86_64' : {
```

Comment:
So no traditional for 15.2?

Reply:
This is via SCC and there will be no traditional. Leap 15.2-uyuni is not yet created. So this one might have traditional.

Reply:
But for SCC the client tools for Leap are those from SLE.

Meaning:
- SLE15SPX will have the tools from SLE15 (SP0/GA)
- openSUSE Leap 15.X will have the tools from openSUSE Leap 15.0

In other words, I can't see a way of avoiding traditional for 15.2 neither at SCC or for Uyuni itself.

Reply:
There are no client tools channel at all for openSUSE when you mirror it via the product page and you do not need it.

Reply:
Right. Salt comes from openSUSE Leap 15.X directly. No idea what I was thinking about.

No more questions, so approving PR :-)